src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -138,8 +140,9 @@ const AnnotationActivity = ({
                         'bcs-is-pending': isPending || error,
                     })}
                 >
-                    <Media.Figure>
+                    <Media.Figure className="bcs-AnnotationActivity-avatar">
                         <Avatar getAvatarUrl={getAvatarUrl} user={createdByUser} />
```

Comment:
It looks like `Avatar` already uses a `Badgeable` component for the "external" icon internally. Can we expand the `Avatar` API slightly to allow a badge to be passed in, instead? That way we don't have to restyle it every time it's used and a mixin wouldn't be necessary.

Reply:
I think this would require slight changes to the border of the badge, at least for the medium size. From what I see in the designs (and code) for medium there should be `2px` border instead of `1px`.

Reply:
I that in this commit https://github.com/box/box-ui-elements/pull/3176/commits/affc7b8925a3dc13612406e384ba9fc43e837e3b

Not exactly sure if that's what you had in mind.
I've added it to storybook and that's how it looks like
(I don't know why the image doesn't show, it's available after click)

![Screenshot 2022-12-15 at 15 45 21](https://user-images.githubusercontent.com/8361509/207890359-bc5f99f8-952e-410a-b093-cf4208b55308.png)

I'm not sure if it's worth the effort, there are slight visual regressions because it was hard to have the same placement for icons of various size.

Reply:
Talked with designers and the decision was to go with badge icon sizes being related to Avatar size, as in design system. Also, I didn't touch the external badge styling.